src/server.js

```diff
@@ -14,13 +16,34 @@ server.connection({
   },
 });
 
-server.register([Inert,], (err) => {
+server.register([ Inert, ], (err) => {
   if(err) throw err;
 
+  server.state('userCookie', {
+    ttl: 360000,
+    isSecure: true,
+    isHttpOnly: false,
+    encoding: 'base64json',
+    clearInvalid: true,
+    strictHeader: true,
+  });
+
   server.route([{
     path: '/',
     method: 'GET',
-    handler: (request, reply) => { reply.file('public/index.html'); },
+    handler: (request, reply) => {
+      const userCookie = request.state.userCookie || '';
+      if(userCookie) {
```

Comment:
! nono, do not do this. I know this is an unfinished pull, but this is dodgy auth code and is a liability.

Do it the hapi way, create an auth strategy, delete this code